src/utils/breakpoint/astBreakpointLocation.js

```diff
@@ -0,0 +1,28 @@
+import { getSymbols } from "../parser";
+import { containsPosition } from "../parser/utils/helpers";
+
+export async function getASTLocation(source, location) {
+  const _source = source.toJS ? source.toJS() : source;
+  const symbols = await getSymbols(_source);
+  const functions = symbols.functions;
+
+  const scope = functions.find(node => {
+    return containsPosition(node.location, location);
+  });
+  if (scope) {
+    const line = location.line - scope.location.start.line;
+    const column = location.column;
+    return { scope, offset: { line, column } };
```

Comment:
hmm 
pros to removing everything but the name: less to save in the persisted data
cons to removing everything but the name: we would have to calculate the parents position to compare it to the new location